### PR TITLE
Add ReCaptcha to campaign pages

### LINKED ISSUE #571: ReCaptcha is missing from campaign pages
Tried making a PayPal donation from https://donate-wagtail.mofostaging.net/fr/fighting-back/, and after processing payment, the page didn’t reload. It looks like ReCaptcha is missing from campaign pages

Capture d’écran 2019-11-11 à 18 33 06

### PR DESCRIPTION
**Pull request by TheoChevalier**
Fixes #571

### CODE REVIEW

**Comment by TheoChevalier** [comment]
Haven’t been able to test, but this should fix it? 
Assuming we need the same scripts as for the landing page

File: donate/templates/pages/core/campaign_page.html [hunk 1]
```diff
@@ -75,4 +75,19 @@ <h1 class="heading heading--primary">
...
+{% endblock %}
+
+{% block extra_js %}
+    {{ currencies|json_script:"currencies" }}
+    {{ braintree_params|json_script:"payments__braintree-params" }}
+    <script src="https://www.paypalobjects.com/api/checkout.min.js" data-version-4></script>
+    <script src="{% static '_js/payments-paypal.compiled.js' %}"></script>
+    {% if recaptcha_site_key %}
+    <script src="https://www.google.com/recaptcha/api.js?render=explicit" async defer></script>
+    {% endif %}
```
Comment (cadecairos) [inline donate/templates/pages/core/campaign_page.html]: We don't need to re-define the extra_js block, we should only need to inclued the g-recaptcha div in the campaign page content block.

Comment (cadecairos) [reply donate/templates/pages/core/campaign_page.html]: (This template inherits from landing_page.html, and so already contains the scripts)

Comment (TheoChevalier) [reply donate/templates/pages/core/campaign_page.html]: aaah, right! Updated

**Pull request merged**